### ISSUE TITLE
I want to increase the view range of tower to avoid excessive barbarian spawns. Which line do I have to change?

### ISSUE THREAD

**Issue by RakibRyan** [ActualSailorCat/Watchtower-Improvement#1]
I tried this but didn't seem to increase the view range. 
INSERT OR REPLACE INTO ModifierArguments
		(ModifierId,									Name,				Value)
VALUES	('SAILOR_WATCHTOWER_MAJOR_SIGHT_MOD',			'Amount',			4),
		('SAILOR_WATCHTOWER_TREELINE_MOD',				'CanSee',			2),
		('SAILOR_WATCHTOWER_MINOR_SIGHT_MOD',			'Amount',			3);

**Comment by ActualSailorCat** [comment]
Don't change the 'CanSee' line. Keep that as 1. But the rest should work as you have it. I assume you're opening Defines.sql, editing, and saving?

**Comment by RakibRyan** [comment]
yes it was from defines.sql  I'm using it with your "Barbarian Outposts to Watchtowers" mod and barbarians clan mode. While the tower gets build but the view range is still the same.

**Comment by ActualSailorCat** [comment]
Check your database log in User\AppData\Local\Firaxis Games\Sid Meier's Civilization VI\Logs for errors.

**Comment by RakibRyan** [comment]
ok it works the game needed a reload. but what I wanted was to change the range of watchtower itself. that I can't get it to work.

**Comment by ActualSailorCat** [comment]
That's tied to how far you see from owned territory. If it's possible to change it, I don't remember how.
